octicons/templatetags/octicons.py

```diff
@@ -102,6 +103,26 @@ def _calculate_height(self, width):
         return int((int(width) * self.height) / self.width)
 
 
+    # Register method for a new icon.
+    @classmethod
+    def register(cls, symbol, svg_file):
```

Comment:
Add a docstring for this method, which should explain what this method does and how it should be used. This is important since it's a public method.